### ISSUE TITLE
I can't execute pulse-dlna

### ISSUE THREAD

**Issue by 3nd0y** [masmu/pulseaudio-dlna#359]
Hi Devs,

i have problem while executing pulseaudio-dlna as decribe below:

```
~/Github/pulseaudio-dlna$ python /usr/bin/pulseaudio-dlna 
Traceback (most recent call last):
  File "/usr/bin/pulseaudio-dlna", line 9, in <module>
    load_entry_point('pulseaudio-dlna==0.5.2', 'console_scripts', 'pulseaudio-dlna')()
  File "/usr/local/lib/python2.7/dist-packages/pulseaudio_dlna-0.5.2-py2.7.egg/pulseaudio_dlna/__main__.py", line 157, in main
  File "/usr/local/lib/python2.7/dist-packages/pulseaudio_dlna-0.5.2-py2.7.egg/pulseaudio_dlna/application.py", line 35, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pulseaudio_dlna-0.5.2-py2.7.egg/pulseaudio_dlna/plugins/chromecast/__init__.py", line 23, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pulseaudio_dlna-0.5.2-py2.7.egg/pulseaudio_dlna/plugins/chromecast/mdns.py", line 23, in <module>
  File "/usr/local/lib/python2.7/dist-packages/zeroconf-0.21.3-py2.7.egg/zeroconf.py", line 175
    def current_time_millis() -> float:
                              ^
SyntaxError: invalid syntax

```

first i build from git, unsuccessful and then try from PPA, also failed. Maybe you have an idea?

**Comment by igorangst** [comment]
Hi, 
I'm not a dev, but I faced the same problem when installing pulseaudio-dlna from the git repository. The problem is actually not with pulseaudio-dlna, but with a python library it uses: zeroconf. The syntax error you get is due to a python version conflict: The line highlighted
`def current_time_millis() -> float:`
contains a type hint, which has been introduced in python 3.5 (see [PEP483](https://www.python.org/dev/peps/pep-0483/)). However, pulseaudio-dlna is running on python 2.7. Looking at the version history of the library in question ([zeroconf](https://pypi.org/project/zeroconf/)), I figured out that they dropped support for python 2.7 in version 0.20.0 (the current version is 0.21.3). The weird thing is that on your system (just like on mine), there is the newest - and incompatible - version installed for python 2.7. 

**"Ok, but how to fix it?"**

I fixed this issue by uninstalling the incompatible python package and installing the latest version still supporting python 2.7:

<code>
sudo pip2.7 uninstall zeroconf

pip2.7 install --user -I zeroconf==0.19.1
</code>

Afterwards, pulseaudio-dlna was finally running on my system :-)
